leases/models.py

```diff
@@ -61,6 +61,18 @@ class Meta:
     def order(self):
         return self.orders.filter(order_type=OrderType.LEASE_ORDER).first()
 
+    @property
+    def has_started(self):
+        return today().date() >= self.start_date
+
+    @property
+    def has_ended(self):
+        return today().date() > self.end_date
```

Comment:
Do these also the current timezone into account?

Reply:
Hmmm 🤔 well this calculations are done only by and for the system, so it should use the timezone set by the app? I'm not 100% sure but I guess yes.

Reply:
For some reason I wasn't sure about that dateutil's today(), but it seems to work correctly, so you are most probably absolutely correct 🙂

Reply:
It has worked so far but I never really thought about it 🙈